main/src/cgeo/geocaching/log/LogCacheActivity.java

```diff
@@ -247,15 +248,48 @@ public void onClick(final View view) {
         });
     }
 
-    private ArrayList<TrackableLog> getSortedTrackables() {
-        final ArrayList<TrackableLog> sortedTrackables = new ArrayList<>(trackables);
-        Collections.sort(sortedTrackables, new Comparator<TrackableLog>() {
+    private void setTrackableSort(int position) {
+        final List<Comparator<TrackableLog>> comparators = new ArrayList<>();
```

Comment:
Create a List of Comparators on each call of this setter? Maybe you find another way to do it.